notifications/smart_reminder_serializers.py

```
@@ -0,0 +1,390 @@
+            'medication_id', 'lab_test_id', 'start_date', 'end_date',
+            'frequency', 'times_per_day', 'days_of_week', 'preferred_times',
+            'status', 'is_adaptive', 'priority', 'is_critical',
+            'auto_schedule', 'use_behavior_analysis'
+        ]
+    
+    def create(self, validated_data):
+        """
+        ایجاد یادآوری با تنظیمات هوشمند
+        """
+        # حذف فیلدهای اضافی
+        auto_schedule = validated_data.pop('auto_schedule', True)
```

Ruff (0.12.2)

Local variable `auto_schedule` is assigned to but never used

Remove assignment to unused variable `auto_schedule`

(F841)